src/third_party/starboard/wpe/shared/player/player_internal.cc

```diff
@@ -457,6 +457,12 @@ enum class MediaType {
 
 constexpr char kClearSamplesKey[] = "fake-key-magic";
 
+#if SB_HAS(NATIVE_AUDIO)
+static constexpr GstStreamVolumeFormat volumeFormat = GST_STREAM_VOLUME_FORMAT_LINEAR;
+#else
+static constexpr GstStreamVolumeFormat volumeFormat = GST_STREAM_VOLUME_FORMAT_CUBIC;
+#endif
+
```

Comment:
I do not think making it dependent on NATIVE AUDIO (which no platform currently uses) is the way to go. You effectively changed it for all platforms AFAICS now. This may be ok to do it for ll platforms but then passing 0-1 values does not look good to me when this is the value in db